From: Julian Ospald <hasufell@gentoo.org>
Date: Sun Sep  8 12:25:19 UTC 2013
Subject: respect our gentoo settings

	this has to be at this place or later
	otherwise some of our settings would be overwritten

--- a/bam.lua
+++ b/bam.lua
@@ -168,6 +168,8 @@
 	-- set some platform specific settings
 	settings.cc.includes:Add("src")
 
+	dofile("gentoo.lua") addSettings(settings)
+
 	if family == "unix" then
 		if platform == "macosx" then
 			settings.link.frameworks:Add("Carbon")
